test/selectors/routerSelector.test.js

```diff
@@ -0,0 +1,42 @@
+import test from 'blue-tape'
+import * as sel from '../../src/selectors/routerSelector'
+import tk from 'timekeeper'
+import routerState from '../helpers/routerState'
+
+const STATE = routerState()
+
+const FIXED_DATE = '2015-12-13'
+
+test('calendarId()', t => {
+  t.equal(sel.calendarId(STATE), 'MI-RUB', 'extracts calendarId from router')
+  t.equal(sel.calendarId(undefined), 'me', 'returns a fallback calendarId for undefined router state')
+  t.end()
+})
+
+test('calendarType()', t => {
+  t.equal(sel.calendarType(STATE), 'courses', 'extracts calendarType from router')
+  t.equal(sel.calendarType(undefined), 'people', 'returns a fallback calendarType for undefined router state')
+  t.end()
+})
+
+test('viewDate()', t => {
+  t.equal(sel.viewDate(STATE), '2015-10-12', 'extracts date from query parameter')
+  const d = Date.parse(FIXED_DATE)
+  tk.freeze(d)
```

Comment:
Would you mind to rename `tk` to e.g. `TK`? Maybe it’s just my problem, but when I saw `tk` I thought that it’s local variable (not import) and wonder where the heck it’s declared.